com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -578,8 +586,17 @@ private void ProcessNetworkBehaviour(TypeDefinition typeDefinition)
                     instructions.Add(processor.Create(OpCodes.Ldftn, method));
                     instructions.Add(processor.Create(OpCodes.Newobj, m_NetworkHandlerDelegateCtor_MethodRef));
                     instructions.Add(processor.Create(OpCodes.Call, m_NetworkManager_ntable_Add_MethodRef));
+
+                    //IF TOOLS
+                    instructions.Add(processor.Create(OpCodes.Ldsfld, m_NetworkManager_rpc_name_table_FieldRef));
+                    instructions.Add(processor.Create(OpCodes.Ldc_I4, unchecked((int)hash)));
+                    //Removing __nhandler from the name of the static handler
```

Comment:
I think this comment should indicate _why_ we're doing this, rather than _what_ we're doing, since the code already tells us the latter.

Reply:
True.

Reply:
I think this comment should not exist but instead `// NetworkManager.__rpc_name_table.Add(HandlerHash, RpcMethodName);` comment should exist on top of this block similar to the previous block.

Reply:
Ahhh sounds good.